bedrock/firefox/templates/firefox/os/index.html

```diff
@@ -85,18 +85,6 @@
         </ul>
       </div>
     </section>
-    <section class="fxos-hub">
-      <div class="content-wrapper">
-        <div class="hub-container">
-          <h2>{{_('Visit the Participation Hub')}}</h2>
-          <p>{{_('Want to make Firefox OS even better? Get all the resources, tools and documentation you need from the Participation Hub.')}}</p>
-          <p><a class="more" rel="external" href="https://firefoxos.mozilla.community">{{_('Check it out')}}</a></p>
-        </div>
-        <div class="hub-image">
-          {{ high_res_img('firefox/os-new/hub.png', {'width': '241', 'height': '261', 'alt':''}) }}
```

Comment:
Looks like you can also delete both `firefox/os-new/hub.png` and `firefox/os-new/hub-high-res.png` as they are no longer being used.